recipes/flyteidl/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "flyteidl" %}
+{% set version = "0.21.24" %}
```

Comment:
The newest version is `0.21.25`. You can go with this one and the bot should update it but just fyi.

Reply:
Thanks for the catch. Must have _just_ updated after I put this together on the 28th.